arches/app/media/js/models/card.js

```diff
@@ -9,7 +9,7 @@ import CardWidgetModel from 'models/card-widget';
 import CardConstraintsViewModel from 'viewmodels/card-constraints';
 import cardComponentLookup from 'card-components';
 
-export default AbstractModel.extend({
+const CardModel = AbstractModel.extend({
```

Comment:
this is changed because `CardModel` is self-referenced inside the component, and needs to be declared for that not to break